distribution/docker-build/Dockerfile

```diff
@@ -1,21 +1,22 @@
-FROM ubuntu:focal AS builder
+FROM ubuntu:jammy AS builder
```

Comment:
Don't make these docker changes as part of the .net 8 migration.

Reply:
It is a part of the .NET 8 update since .NET 8 is not supported on other version of Ubuntu than >22.04
All of the 20.04 and older distribution or not supported by .NET 8 

https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions

Reply:
I understand, but these docker containers were set up for testing mono which isn't necessary and can be upgraded independently of the .net 8 upgrade.

Reply:
So if i understand everything right, you want me make a PR just for the .NET 8 update on csproj files and other but don't touch the docker file or else related to mono ?

Reply:
No, just remove these changes from this PR.